src/configs/recommended.ts

```diff
@@ -9,6 +9,9 @@ import { jsonConfigs } from './json';
 import { tsFileExtensions, tsRules, type TSRuleExtensions } from './typescript';
 import { getFiles } from './utils';
 
+// @ts-ignore -- untyped
```

Comment:
Personally not a huge fan of this style and I'd prefer something like
```suggestion
// @ts-ignore because it's untyped
```

But in any case I'd prefer declaring a module and creating a small `eslint-plugin-unicorn.d.ts` instead of `ts-ignore`ing it.